react/features/base/conference/actions.js

```diff
@@ -584,6 +584,16 @@ export function dataChannelOpened() {
     };
 }
 
+/**
+ * Action to end a conference for all participants.
+ *
+ * @returns {Function}
+ */
+export function endConference() {
+    return async (dispatch: Dispatch<any>, getState: Function) =>
+        getCurrentConference(getState).end();
```

Comment:
I think this only works if you have joined the conference, so I'd say we should extract the `conference` field from the state. The current conference could be in the joining state.

Reply:
Fixed.